### PR TITLE
Script the Node.js and NPM setup

### PR DESCRIPTION
**Pull request by benbc**

### CODE REVIEW

File: justfile [hunk 1]
```diff
@@ -13,14 +13,27 @@ dev-config:
...
+setup: pip-install npm-install migrate ensure-superuser ensure-outputs collectstatic
 
 # install correct versions of all Python dependencies
 pip-install:
     pip install pip-tools
     pip-sync requirements.txt requirements.dev.txt
     pre-commit install
 
+# install all JS dependencies
+npm-install: check-fnm
```
Comment (rebkwok) [inline justfile:25]: What if you're on windows and not using `fnm`?

**Review by rebkwok — APPROVED** [review]
LGTM, just one question on the fnm-check bit

Also, do we need any command or instructions on updating npm packages if we need to?  It'd just be `npm run build` on its own, right?

**Comment by tomodwyer** [comment]
> Also, do we need any command or instructions on updating npm packages if we need to? It'd just be `npm run build` on its own, right?

When the `package.json` is updated, you would need to run `npm install` then `npm run build`

File: justfile (hunk 1 above)
Comment (benbc) [reply justfile:25]: I thought we could worry about that when it happens. It's unlikely to be needed in practice and anything that we write now won't get continuously tested by someone using it as the rest of the stuff evolves, so it's likely to rot.

**Pull request merged**